wrims-comparison-test/README.md

```diff
@@ -9,5 +9,13 @@ this directory with the following command:
 The testReport task will download a test watershed as a zip artifact from GitHub packages, and run WRIMS in 
 build/testProjects/CalLite4.1_TR with run controls set in the file Test_01.config.
 
+The report that the comparison tool produces is also stored as an artifact. The latest version can be found 
+by following this link on the WRIMS project home page:  
+![image](https://github.com/user-attachments/assets/7ad74689-67fc-4145-a788-4d8591597544)
```

Comment:
These should be more or less static links to capture the current state.

Reply:
Revised to point at .png images local to the project